Automattic/wc-calypso-bridge
# Updated payment gateways info notice for free trial users

### Changes proposed in this Pull Request:



This PR updates the message that is shown for free trial users who set up payments in a country that is not supported by Woo Payments. 

The message is changed from: 

> Only Administrators and Store Managers can place orders during the free trial. If you are ready to accept payments from customers, upgrade to a paid plan.

to:

> Only Administrators and Store Managers can place orders during the free trial. Upgrade to a paid plan to accept payments from customers and explore more payment options.

The new message was suggested by @lauroraa in this Slack thread: p1703167185395439-slack-C03Q87XT1QF. Thanks Laura! :)

Closes #1346






### How to test the changes in this Pull Request:




1. Create a new Woo Express free trial development site.
2. During the setup process, select a country that doesn't support Woo Payments, like South Africa. If you have already created the site, change the country from **Settings > WooCommerce > General**.
3. Go to the My Home tab.
4. Click on the "Set up payments" task list item. 
5. Once redirected, confirm that the message that shows up at the top of the page reads: 

> Only Administrators and Store Managers can place orders during the free trial. Upgrade to a paid plan to accept payments from customers and explore more payment options.



### Other information:




### FOR PR REVIEWER ONLY:
